code/modules/mob/living/carbon/human/life.dm

```diff
@@ -365,7 +365,7 @@
 	else
 		bodytemperature += (BODYTEMP_HEATING_MAX + (fire_stacks * 12))
 		var/datum/antagonist/vampire/V = mind?.has_antag_datum(/datum/antagonist/vampire)
-		if(V?.get_ability(/datum/vampire_passive/full) && stat != DEAD)
+		if(V && V.get_ability(/datum/vampire_passive/full) && stat != DEAD)
```

Comment:
Put a comment explaining this doenst work.

Cocky people in code review will try and scold you otherwise.

Reply:
Will do later

Reply:
Is there any reason it returns `FALSE` as opposed to null?

Reply:
The proc is stated as a question. So you could expect it to return a boolean